lametro/views.py

```diff
@@ -90,6 +90,9 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         bill = self.get_object()
 
+        sources = list(bill.sources.all())
+        context['sources'] = sources
```

Comment:
If you have access to `bill` in the template, do you need to add sources as an additional context variable, or can you access it via `bill`?

Reply:
Ah gotcha, we already have a way to access the sources with the variables we have, so it makes sense to just use what we've got instead of adding more vars. I've started this change for the bills and I'll hopefully have the rest done by Monday!